content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -196,6 +196,109 @@ add a brief description in the :guilabel:`Description` field. Click :guilabel:`F
+can be accomplished by going to the :guilabel:`Search bar` and typing in `windows defender`.
+
+Next, click :guilabel:`Advanced settings` in the left pane.
+
+.. image:: windows_iot/advanced-settings.png
+   :align: center
+   :alt: Advanced settings option highlighted in the left pane of the Windows Defender Firewall app.
+
+In the left pane, choose :guilabel:`Inbound Rules`.
+
+.. image:: windows_iot/inbound-rules.png
+   :align: center
+   :alt: Windows Defender left window pane with inbound rules menu item highlighted.
+
+After selecting :guilabel:`Inbound Rules`, select :guilabel:`New Rule` in the far right pane.
+
+.. image:: windows_iot/new-rule.png
+   :align: center
+   :alt: New rule dropdown shown with new rule option highlighted.
+
+Then, for the :guilabel:`Rule Type`, select the radio button for :guilabel:`Port`. Click
+:guilabel:`Next` to continue to the rest of the configuration.
+
+.. image:: windows_iot/radio-port.png
+   :align: center
+   :alt: Rule Type window open, with the radio button next to port highlighted.
+
+On the :guilabel:`Protocols and Ports` page, choose the radio button for :guilabel:`TCP` under
+:guilabel:`Does this rule apply to TCP or UDP?`. Next, under :guilabel:`Does this rule apply to all
+local ports or specific ports?`, select the radio button for :guilabel:`Specific local ports`. Enter
+`9050` and click :guilabel:`Next` to continue.
+
+.. image:: windows_iot/protocol-port.png
+   :align: center
+   :alt: Protocol/port configuration window with TCP, specific port (9050) and Next highlighted.
+
+The next screen is the :guilabel:`Action` page. Under :guilabel:`What action should be taken when a
+connection matches the specified conditions?`, choose the radio button for :guilabel:`Allow the
+connection`. Click :guilabel:`Next` to continue.
+
+A :guilabel:`Profile` page appears. Under :guilabel:`When does this rule apply?`, leave the
+three boxes checked for: :guilabel:`Domain`, :guilabel:`Private`, and :guilabel:`Public`. Click
+:guilabel:`Next` to continue to the naming convention page.
+
+On the :guilabel:`Name` page, enter `Odoo Worldline` under the :guilabel:`Name` field. Enter a
+:guilabel:`Description (optional)`. Finally, once ready, click :guilabel:`Finish`.
+
+The final :guilabel:`Inbound rule` should appear as follows:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Name
+     - Profile
+     - Enabled
+     - Action
+     - Override
+     - Program
+     - Local Address
+     - Remote Address
+     - Protocol
+     - Local Port
+     - Remote Port
+     - Authorized Users
+     - Authorized Computers
+     - Authorized Local Principals
+     - Local User Owner
+     - PolicyAppld
+     - Application Package
+   * - Odoo Worldline
+     - All
+     - Yes
+     - Allow
+     - No
+     - Any
+     - Any
+     - Any
+     - TCP
+     - 9050
+     - Any
+     - Any
+     - Any
+     - Any
+     - Any
+     - None
+     - Any
```

Comment:
you may need to speak with SALI about how to properly format this, as it is showing up a bit distorted/overlapping in my run through. Here's a screenshot of what I'm seeing...
https://drive.google.com/file/d/1_ZztD0ubjuiQf3ySLhumICpcAK--4ZG_/view?usp=sharing

Reply:
@samueljlieber 

When you have a moment can you take a look at this render? I would like to have the full table in this doc, if at all possible.

Reply:
![2024-02-20_14-16](https://github.com/odoo/documentation/assets/91896312/8e4c1cac-70c4-4490-a5b0-273cab780af3)

I'm not showing this error when rendered and opened in Chrome.

@ksc-odoo  What browser are you viewing this in?

Reply:
I use Chrome, as well. If @samueljlieber can confirm that there is no issue with the formatting, you can proceed to push this to the next stage.

Reply:
Hey @tiku-odoo, when a table has too many columns it can end up overlapping/overflowing on the page because a table will only reduce its width to the contents of the cells. 

To avoid this, you can flip the axis of the table so there are many rows instead of many columns:
<img width="698" alt="Screenshot 2024-02-20 at 2 26 00 PM" src="https://github.com/odoo/documentation/assets/36018073/ed2faf91-b8cd-473a-870d-fe9b473f2c93">

```suggestion
.. list-table::
   :header-rows: 1
   :stub-columns: 1

   * - 
     - Odoo Worldline
   * - Profile
     - All
   * - Enabled
     - Yes
   * - Action
     - Allow
   * - Override
     - No
   * - Program
     - Any
   * - Local Address
     - Any
   * - Remote Address
     - Any
   * - Protocol
     - TCP
   * - Local Port
     - 9050
   * - Remote Port
     - Any
   * - Authorized Users
     - Any
   * - Authorized Computers
     - Any
   * - Authorized Local Principals
     - Any
   * - Local User Owner
     - Any
   * - PolicyAppld
     - None
   * - Application Package
     - Any
```